java/src/main/java/com/adhoc/flight/utils/PrintTinyIntVectorProcessor.java

```diff
@@ -0,0 +1,65 @@
+package com.adhoc.flight.utils;
+
+import org.apache.arrow.vector.TinyIntVector;
+
+import com.adhoc.flight.client.BaseTinyIntVectorProcessor;
+
+public final class PrintTinyIntVectorProcessor extends BaseTinyIntVectorProcessor {
+
+	private long count;
+	
+	@Override
+	public long getCount() {
+		return count;
+	}
+
+	@Override
+	public void incrementCount() {
+		count++;
+	}
+
+	@Override
+	public String process(TinyIntVector tinyIntVector) {
+		byte[] bytes = new byte[tinyIntVector.getValueCount()];
+		
+		for (int index = 0; index < bytes.length; index++) {
+			bytes[index] = tinyIntVector.getObject(index);
+		}
+		
+		return new String(bytes);
```

Comment:
This doesn't create a string representation of the byte array. It decodes the byte array as a series of code units in the default encoding.

Reply:
Solved in the new version.